staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -80,6 +81,9 @@ var (
 		# Create a copy of mypod adding a debug container and attach to it
 		kubectl debug mypod -it --image=busybox --copy-to=my-debugger
 
+		# Create a copy of mypod adding a debug container and attach to it also add NET_ADMIN, SYS_PTRACE capabilities
+		kubectl debug mypod -it --capabilities=NET_ADMIN,SYS_PTRACE --image=busybox --copy-to=my-debugger
+
```

Comment:
thx,done

Reply:
I share the same concern as Jordan, I don't want `kubectl debug` to end up just like `kubectl run` with too many flags which only complicate their use. For now I'd like to hear how many use-cases are for such granular flags. I'm inclined to do something like `--overrides` in `kubectl run` for those cases rather than adding more and more flags. 
/hold